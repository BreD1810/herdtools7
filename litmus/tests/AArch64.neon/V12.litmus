AArch64 V12
(* Tests LD1R, no offset *)
{
x=1;
0:X0=x;0:X1=y; 0:X2=z; 0:X3=a;
}

 P0                     ;
 LD1R {V0.2S}, [X0]     ;
 ST1 {V0.S}[0], [X1]    ;
 ST1 {V0.S}[1], [X2]    ;
 ST1 {V0.S}[2], [X3]    ;
 forall (y=1 /\ z=1 /\ a=0)
