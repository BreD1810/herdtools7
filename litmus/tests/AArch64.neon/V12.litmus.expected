Test V12 Required
Histogram (1 states)
1000000:>a=0; y=1; z=1;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=1 /\ z=1 /\ a=0) is validated
Hash=6b9a2139773a87cbf79374d1f01d9621
Observation V12 Always 1000000 0
